### DISCUSSION TITLE
Extra Padding Around Clock

### DISCUSSION THREAD

**Discussion by TechStudent10** [martinet101/ElevenClock#465] [Q&A] [answered]
Is there a way to add more padding around the clock itself? This just kinda looks weird to me.

![image](https://user-images.githubusercontent.com/76978184/148550055-b7d47b59-d4ac-46c9-ae21-e3322157e86c.png)

**Comment by marticliment** [comment, marked as answer]
Not for the moment. but there's a Feature request about custom size and position, what could, in some way, fix that

**Reply by TechStudent10** [reply]
Hm, alright.
